api/cities-api/index.js

```diff
@@ -0,0 +1,16 @@
+/* eslint-disable no-console */
+
+const express = require('express');
+const app = express();
+const PORT = 3002;
+const path = require('path');
+
+app.get('/', (req, res) => {
+  res.send('Home');
+});
+
+app.use('/cities', express.static(path.join(__dirname, 'cities.json')));
```

Comment:
This will return the JSON file content as a string.
Is there a way we can fix this?